examples/sign-leaf-with-ca.rs

```diff
@@ -0,0 +1,99 @@
+use rcgen::{
+	date_time_ymd, BasicConstraints, Certificate, CertificateParams, CertificateSigningRequest,
+	DnType, ExtendedKeyUsagePurpose, IsCa, KeyUsagePurpose,
+};
+use time::{Duration, OffsetDateTime};
+use x509_parser::certification_request::X509CertificationRequest;
+use x509_parser::prelude::FromDer;
+
+/// Example demonstrating signing end-endity certificate with ca
+fn main() {
+	let ca = Ca::new();
+	let entity = Entity::new();
+
+	println!("directly signed end-entity certificate:");
+	let direct = entity
+		.certificate
+		.serialize_pem_with_signer(&ca.certificate)
+		.unwrap();
+	println!("{direct}");
+
+	println!("ca certificate:");
+	let pem = ca.certificate.serialize_pem().unwrap();
+
+	println!("{}", pem);
```

Comment:
Same feedback here RE: re-arranging for one `println!`.

If you keep the separate `println!`'s I think this second instance should use an in-line var like `println!("{pem}")` for consistency with L19.

I also think `ca_cert_pem` might be a better name for the var as opposed to `pem`.